app/controllers/comments/approvedBy.js

```diff
@@ -0,0 +1,66 @@
+import { db } from "~/utils/db.server";
+import { approvedByCommentSchema } from "~/utils/backend/validators/comments";
+import { INVALID_PARAMS_FOR_OPERATION_ERROR_MESSAGE } from "~/utils/constants";
+import { COMMENT_AS_AN_ANSWER, DEFAULT_ERROR_MESSAGE, UPDATE_COMMENT_ERROR_MESSAGE} from '~/utils/backend/constants';
```

Comment:
Remove/use unused variable: 

`'UPDATE_COMMENT_ERROR_MESSAGE'` is defined but never used`

Reply:
Done